ajax/libs/fullcalendar-scheduler/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "fullcalendar-scheduler",
+  "filename": "scheduler.min.js",
+  "version": "1.9.0",
+  "description": "A premium add-on to FullCalendar for displaying events and resources",
+  "author": {
+    "name": "Adam Shaw",
+    "email": "arshaw@arshaw.com",
+    "homepage": "http://arshaw.com/"
+  },
+  "homepage": "https://fullcalendar.io/scheduler/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fullcalendar/fullcalendar-scheduler.git"
+  },
+  "keywords": [
+    "calendar",
+    "event",
+    "full-sized",
+    "jquery-plugin"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fullcalendar/fullcalendar-scheduler.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
Let's use the pattern `**/!(*.ts)` because the current pattern would grab `scheduler.d.ts` which can't be used in web front-end in v1.9.1~v1.9.2.
Many thanks.